corehq/apps/userreports/reports/builder/__init__.py

```diff
@@ -142,6 +147,14 @@ def make_owner_name_indicator(column_id):
     }
 
 
+def make_user_name_indicator(column_id):
+    return _make_user_group_or_location_indicator("user_id", column_id)
+
+
+def make_owner_name_indicator(column_id):
+    return _make_user_group_or_location_indicator("owner_id", column_id)
```

Comment:
OK

On 12 November 2015 at 16:55, Noah Carnahan <notifications@github.com>
wrote:

> In corehq/apps/userreports/reports/builder/__init__.py
> <https://github.com/dimagi/commcare-hq/pull/8701#discussion_r44665476>:
>
> > @@ -142,6 +147,14 @@ def make_owner_name_indicator(column_id):
> >      }
> >
> >
> > +def make_user_name_indicator(column_id):
> > +    return _make_user_group_or_location_indicator("user_id", column_id)
> > +
> > +
> > +def make_owner_name_indicator(column_id):
> > +    return _make_user_group_or_location_indicator("owner_id", column_id)
>
> You're right they aren't B3 related. I made some report builder bug fixes
> from the B3 QA process that I just added straight to this PR to avoid
> additional staging madness.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8701/files#r44665476>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa